bazel_configure.py

```diff
@@ -54,19 +54,19 @@ def discover_timestamp():
 # Get the processor the platform is running on
 ######################################################################
 def discover_processor():
-  return platform.uname().machine
+  return platform.uname()[4]
 
 ######################################################################
 # Get the operating system of the platform
 ######################################################################
 def discover_os():
-  return platform.uname().system
+  return platform.uname()[0]
 
 ######################################################################
 # Get the operating system version
 ######################################################################
 def discover_os_version():
-  return platform.uname().release
+  return platform.uname()[2]
```

Comment:
However good point, I'm replacing the calls as @nlu90 suggest, except that these are function calls, not attributes.